Deprecate Animation and Slider components (#802)
vtex-apps/store-components: docs/Animation.md

# Animation ![https://img.shields.io/badge/-Deprecated-red](https://img.shields.io/badge/-Deprecated-red)

## Description

> :warning: **The Product Price block has been deprecated**. Although support for this block is still granted.

Reviewer: **The Product Price block has been deprecated**, although support for this block is still given.

> :warning: **The Product Price block has been deprecated**. Although support for this block is still granted.
> 

Reviewer 2: ```suggestion
:warning: **The Animation block has been deprecated** (support is still granted).

```

Reviewer: :warning: **The Animation block has been deprecated** (support is still given).
